packages/common/src/store/buy-usdc/selectors.ts

```diff
@@ -0,0 +1,12 @@
+import { CommonState } from 'store/reducers'
+
+export const getBuyUSDCProvider = (state: CommonState) => state.buyUSDC.provider
+
+export const getBuyUSDCFlowStage = (state: CommonState) => state.buyUSDC.stage
+
+export const getBuyUSDCFlowError = (state: CommonState) => state.buyUSDC.error
```

Comment:
Doesn't look like we use this anywhere yet. Is there a plan to surface this? I think all the UI cares about is whether there is an error and what stage it happened in, not necessarily what the error was (that should be captured and reported in the saga already anyway)

Reply:
Correct. I _might_ use it when I add error handling to the purchase content flow. But likely I will bubble the usdc error up into the purchase content saga and set the error in that slice.